cmd/clusterctl/examples/openstack/provider-component/user-data/ubuntu/templates/worker-user-data.sh

```diff
@@ -20,7 +21,7 @@ apt-get update
 apt-get install -y apt-transport-https prips
 apt-key adv --keyserver hkp://keyserver.ubuntu.com --recv-keys F76221572C52609D
 cat <<EOF > /etc/apt/sources.list.d/k8s.list
-deb [arch=amd64] https://apt.dockerproject.org/repo ubuntu-xenial main
+deb [arch=amd64] https://apt.dockerproject.org/repo ubuntu-${DISTRIBUTOR} main
```

Comment:
Ubuntu 18.04 lsb_release -cs returns bionic, but there is no kubernetes-bionic: https://apt.dockerproject.org/repo/dists/

Reply:
you are right, for bionic I tested and it works with ubuntu-xenial 

so seems this should not be a good PR now, thanks for bring this out @hidekazuna